akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -108,25 +108,10 @@ final class Source[+Out, +Mat](private[stream] override val module: Module)
    * Concatenates a second source so that the first element
    * emitted by that source is emitted after the last element of this
    * source.
-   */
-  def concat[Out2 >: Out, M](second: Graph[SourceShape[Out2], M]): Source[Out2, (Mat, M)] = concatMat(second)(Keep.both)
-
-  /**
-   * Concatenates a second source so that the first element
-   * emitted by that source is emitted after the last element of this
-   * source.
-   */
-  def concatMat[Out2 >: Out, Mat2, Mat3](second: Graph[SourceShape[Out2], Mat2])(
-    combine: (Mat, Mat2) ⇒ Mat3): Source[Out2, Mat3] = Source.concatMat(this, second)(combine)
-
-  /**
-   * Concatenates a second source so that the first element
-   * emitted by that source is emitted after the last element of this
-   * source.
    *
    * This is a shorthand for [[concat]]
    */
-  def ++[Out2 >: Out, M](second: Graph[SourceShape[Out2], M]): Source[Out2, (Mat, M)] = concat(second)
+  def ++[Out2 >: Out, M](second: Graph[SourceShape[Out2], M]): Source[Out2, Mat] = concat(second)
```

Comment:
If `concat` is on `FlowOps` and `++` is an alias for it, then I think it should go on `FlowOps`, at least that makes sense to me.